test/data/20180905A.x9

```diff
@@ -0,0 +1,23 @@
+0135T231380104121042882201810012313NCitadel           Wells Fargo        US     
```

Comment:
FYI, directories named `testdata/` are skipped by the go tool.

```
$ go help test
..
The go tool will ignore a directory named "testdata", making it available
...
```

Reply:
That was the requirement for ACH.  Are you saying we don't want to have a testdata directory for x9? or are you saying we should eliminate testdata from both libraries.

Reply:
Test files are totally fine with me. I was just mentioning the standard Go convention is to put them in a `testdata/` directory.

Reply:
I was following this, https://github.com/golang-standards/project-layout, which only had a test directory.  Is that not correct?

Reply:
Oh. I didn't realize that project has `test/`. Either is fine with me. 

There's an open issue on that project https://github.com/golang-standards/project-layout/issues/3